### PR TITLE
Change metrics.Gauge data type to float64

### PR DESCRIPTION
**Pull request by peterbourgon**
Does what it says on the tin :)

### CODE REVIEW

**Comment by daniel-garcia** [comment]
:+1:

File: metrics/expvar/expvar.go [hunk 1]
```diff
@@ -100,7 +100,7 @@ func (h *histogram) Observe(value int64) {
 	}
 
 	for q, gauge := range h.gauges {
-		gauge.Set(h.hist.Current.ValueAtQuantile(float64(q)))
+		gauge.Set(float64(h.hist.Current.ValueAtQuantile(float64(q))))
```
Comment (tsenart) [inline metrics/expvar/expvar.go]: The inner conversion of `q` to `float64` becomes unnecessary.

Comment (tsenart) [inline metrics/expvar/expvar.go]: Oh, never mind. You kept the key type in the `gauges` map.

**Comment by mreiferson** [comment]
remember when fixed point was cool?

(also LGTM)

**Pull request merged**